### ISSUE TITLE
Several Services stop at startup

### ISSUE THREAD

**Issue by franciscoRdiaz** [elastest/elastest-device-emulator-service#77]
Hi Varun,

When I try to startup the EDS both from "Docker Toolbox for windows or from ETM GUI", a few seconds afther starting, all services but one are stopped.

I send you the logs of those services:

- elastest/eds-frontend:0.5.0-alpha1:
`PYTHONPATH: :/home/elastest/eds/FrontEnd/openmtc-gevent/../futile/src:/home/elastest/eds/FrontEnd/openmtc-gevent/../common/openmtc/lib:/home/elastest/eds/FrontEnd/openmtc-gevent/../common/openmtc-etsi/src:/home/elastest/eds/FrontEnd/openmtc-gevent/../common/openmtc-onem2m/src:/home/elastest/eds/FrontEnd/openmtc-gevent/../common/openmtc/src:/home/elastest/eds/FrontEnd/openmtc-gevent/../serializers/*/src:/home/elastest/eds/FrontEnd/openmtc-gevent/../openmtc-app/src
PYTHONPATH: :/home/elastest/eds/FrontEnd/openmtc-gevent/../futile/src:/home/elastest/eds/FrontEnd/openmtc-gevent/../common/openmtc/lib:/home/elastest/eds/FrontEnd/openmtc-gevent/../common/openmtc-etsi/src:/home/elastest/eds/FrontEnd/openmtc-gevent/../common/openmtc-onem2m/src:/home/elastest/eds/FrontEnd/openmtc-gevent/../common/openmtc/src:/home/elastest/eds/FrontEnd/openmtc-gevent/../serializers/*/src:/home/elastest/eds/FrontEnd/openmtc-gevent/../openmtc-app/src:/home/elastest/eds/FrontEnd/openmtc-gevent/src:/home/elastest/eds/FrontEnd/openmtc-gevent/../server/openmtc-cse/src:/home/elastest/eds/FrontEnd/openmtc-gevent/../server/openmtc-ngsi/src:/home/elastest/eds/FrontEnd/openmtc-gevent/../server/openmtc-scl/src:/home/elastest/eds/FrontEnd/openmtc-gevent/../server/openmtc-server/src
PYTHONPATH: :/home/elastest/eds/FrontEnd/../futile/src:/home/elastest/eds/FrontEnd/../common/openmtc/lib:/home/elastest/eds/FrontEnd/../common/*/src:/home/elastest/eds/FrontEnd/../serializers/*/src:/home/elastest/eds/FrontEnd/../openmtc-app/src
PYTHONPATH: :/home/elastest/eds/FrontEnd/openmtc-app/src
DEBUG:openmtc_gevent.main:Reading config file: config-gscl.json
DEBUG:__main__:Trying config file location: /home/elastest/eds/FrontEnd/config.json
INFO:__main__:Reading configuration file /home/elastest/eds/FrontEnd/config.json.
INFO:engineio:Server initialized for gevent.
INFO:FrontEnd:Registering application as FrontEnd.
DEBUG:OneM2MHTTPClient:Mapping OneM2M request to generic request: OneM2MRequest: originator: CFrontEnd | request_identifier: bawN6Hk8es9Tf314 | originating_timestamp: None | filter_criteria: None | request_expiration_timestamp: None | result_persistence: None | result_content: None | discovery_result_type: None | group_request_identifier: None | content: AE(path='None', id='None') | to: onem2m | delivery_aggregation: None | role: None | response_type: None | result_expiration_timestamp: None | operation: OneM2MOperation.create | operation_execution_time: None | resource_type: <class 'openmtc_onem2m.model.AE'> | event_category: None
DEBUG:openmtc_onem2m.serializer.util:Encoding result: AE(path='None', id='None') - application/json
DEBUG:OneM2MJsonSerializer:Encoding representation: {'ontologyRef': None, 'App-ID': u'dummy', 'appName': None, 'resourceType': None, 'accessControlPolicyIDs': [], 'labels': [], 'announcedAttribute': None, 'creationTime': None, 'announceTo': None, 'contentSerialization': None, 'pointOfAccess': [u'http://127.0.0.1:6065'], 'requestReachability': False, 'parentID': None, 'lastModifiedTime': None, 'resourceName': u'FrontEnd', 'expirationTime': datetime.datetime(2017, 10, 5, 11, 36, 0, 975291, tzinfo=<openmtc.util.Utc object at 0x7f9827e152c0>), 'resourceID': None, 'childResource': [], 'AE-ID': None, 'nodeLink': None}
DEBUG:OneM2MHTTPClient:Added request params: {}
ERROR:FlaskRunner:Error
Traceback (most recent call last):
  File "build/bdist.linux-x86_64/egg/openmtc_app/runner/__init__.py", line 18, in run
    self._run()
  File "build/bdist.linux-x86_64/egg/openmtc_app/flask_runner/__init__.py", line 49, in _run
    self.m2m_app.run(self, self.m2m_ep)
  File "build/bdist.linux-x86_64/egg/openmtc_app/onem2m.py", line 113, in run
    self.register()
  File "build/bdist.linux-x86_64/egg/openmtc_app/onem2m.py", line 191, in register
    registration = self.create_application(app)
  File "build/bdist.linux-x86_64/egg/openmtc_app/onem2m.py", line 286, in create_application
    app = self.mapper.create(path, application)
  File "build/bdist.linux-x86_64/egg/openmtc_onem2m/mapper/__init__.py", line 47, in create
    pc=instance
  File "build/bdist.linux-x86_64/egg/aplus/__init__.py", line 128, in get
    raise self.reason
ConnectionFailed: Error during HTTP request: [Errno 111] Connection refused. Request was: POST http://localhost:8000/onem2m (0.0679s)
DEBUG:FlaskRunner:Shutting down
DEBUG:FrontEnd:shutdown handler is finished
DEBUG:FrontEnd:subscriptions deleted
DEBUG:FrontEnd:app deleted
Traceback (most recent call last):
  File "/usr/lib/python2.7/runpy.py", line 174, in _run_module_as_main
    "__main__", fname, loader, pkg_name)
  File "/usr/lib/python2.7/runpy.py", line 72, in _run_code
    exec code in run_globals
  File "/home/elastest/eds/FrontEnd/src/frontend/__main__.py", line 32, in <module>
    Runner(app, host=host, port=port).run(ep)
  File "build/bdist.linux-x86_64/egg/openmtc_app/runner/__init__.py", line 18, in run

  File "build/bdist.linux-x86_64/egg/openmtc_app/flask_runner/__init__.py", line 49, in _run

  File "build/bdist.linux-x86_64/egg/openmtc_app/onem2m.py", line 113, in run
  File "build/bdist.linux-x86_64/egg/openmtc_app/onem2m.py", line 191, in register
  File "build/bdist.linux-x86_64/egg/openmtc_app/onem2m.py", line 286, in create_application
  File "build/bdist.linux-x86_64/egg/openmtc_onem2m/mapper/__init__.py", line 47, in create

  File "build/bdist.linux-x86_64/egg/aplus/__init__.py", line 128, in get

openmtc.exc.ConnectionFailed: Error during HTTP request: [Errno 111] Connection refused. Request was: POST http://localhost:8000/onem2m (0.0679s)`

- elastest/eds-zigbeeipe:0.5.0-alpha1
`PYTHONPATH: :/home/elastest/eds/ZigBeeIPE/../futile/src:/home/elastest/eds/ZigBeeIPE/../common/openmtc/lib:/home/elastest/eds/ZigBeeIPE/../common/*/src:/home/elastest/eds/ZigBeeIPE/../serializers/*/src:/home/elastest/eds/ZigBeeIPE/../openmtc-app/src
PYTHONPATH: :/home/elastest/eds/ZigBeeIPE/openmtc-app/src
zig-bee-ipe: 8: zig-bee-ipe: cannot open /dev/ttyACM0: No such file
DEBUG:__main__:Trying config file location: /home/elastest/eds/ZigBeeIPE/config.json
INFO:__main__:Reading configuration file /home/elastest/eds/ZigBeeIPE/config.json.
INFO:ZigBeeIPE:Registering application as ZigBeeIPE.
DEBUG:OneM2MHTTPClient:Mapping OneM2M request to generic request: OneM2MRequest: originator: CZigBeeIPE | request_identifier: HFUz6doP5KV8SqNt | originating_timestamp: None | filter_criteria: None | request_expiration_timestamp: None | result_persistence: None | result_content: None | discovery_result_type: None | group_request_identifier: None | content: AE(path='None', id='None') | to: onem2m | delivery_aggregation: None | role: None | response_type: None | result_expiration_timestamp: None | operation: OneM2MOperation.create | operation_execution_time: None | resource_type: <class 'openmtc_onem2m.model.AE'> | event_category: None
DEBUG:openmtc_onem2m.serializer.util:Encoding result: AE(path='None', id='None') - application/json
DEBUG:OneM2MJsonSerializer:Encoding representation: {'ontologyRef': None, 'App-ID': u'dummy', 'appName': None, 'resourceType': None, 'accessControlPolicyIDs': [], 'labels': [], 'announcedAttribute': None, 'creationTime': None, 'announceTo': None, 'contentSerialization': None, 'pointOfAccess': [u'http://127.0.0.1:5001'], 'requestReachability': False, 'parentID': None, 'lastModifiedTime': None, 'resourceName': u'ZigBeeIPE', 'expirationTime': datetime.datetime(2017, 10, 5, 11, 36, 8, 329680, tzinfo=<openmtc.util.Utc object at 0x7f8ec1f1d2e0>), 'resourceID': None, 'childResource': [], 'AE-ID': None, 'nodeLink': None}
DEBUG:OneM2MHTTPClient:Added request params: {}
ERROR:SimpleFlaskRunner:Error
Traceback (most recent call last):
  File "build/bdist.linux-x86_64/egg/openmtc_app/runner/__init__.py", line 18, in run
    self._run()
  File "build/bdist.linux-x86_64/egg/openmtc_app/flask_runner/__init__.py", line 49, in _run
    self.m2m_app.run(self, self.m2m_ep)
  File "build/bdist.linux-x86_64/egg/openmtc_app/onem2m.py", line 113, in run
    self.register()
  File "build/bdist.linux-x86_64/egg/openmtc_app/onem2m.py", line 191, in register
    registration = self.create_application(app)
  File "build/bdist.linux-x86_64/egg/openmtc_app/onem2m.py", line 286, in create_application
    app = self.mapper.create(path, application)
  File "build/bdist.linux-x86_64/egg/openmtc_onem2m/mapper/__init__.py", line 47, in create
    pc=instance
  File "build/bdist.linux-x86_64/egg/aplus/__init__.py", line 128, in get
    raise self.reason
ConnectionFailed: Error during HTTP request: [Errno -2] Name or service not known. Request was: POST http://frontend:8000/onem2m (0.0102s)
DEBUG:SimpleFlaskRunner:Shutting down
INFO:ZigBeeCoordinator:Shutting down.
DEBUG:ZigBeeIPE:shutdown handler is finished
DEBUG:ZigBeeIPE:subscriptions deleted
DEBUG:ZigBeeIPE:app deleted
Traceback (most recent call last):
  File "/usr/lib/python2.7/runpy.py", line 174, in _run_module_as_main
    "__main__", fname, loader, pkg_name)
  File "/usr/lib/python2.7/runpy.py", line 72, in _run_code
    exec code in run_globals
  File "/home/elastest/eds/ZigBeeIPE/src/zigbeeipe/__main__.py", line 45, in <module>
    Runner(app, host=host, port=port).run(ep)
  File "build/bdist.linux-x86_64/egg/openmtc_app/runner/__init__.py", line 18, in run

  File "build/bdist.linux-x86_64/egg/openmtc_app/flask_runner/__init__.py", line 49, in _run

  File "build/bdist.linux-x86_64/egg/openmtc_app/onem2m.py", line 113, in run
  File "build/bdist.linux-x86_64/egg/openmtc_app/onem2m.py", line 191, in register
  File "build/bdist.linux-x86_64/egg/openmtc_app/onem2m.py", line 286, in create_application
  File "build/bdist.linux-x86_64/egg/openmtc_onem2m/mapper/__init__.py", line 47, in create

  File "build/bdist.linux-x86_64/egg/aplus/__init__.py", line 128, in get

openmtc.exc.ConnectionFailed: Error during HTTP request: [Errno -2] Name or service not known. Request was: POST http://frontend:8000/onem2m (0.0102s)`

- elastest/eds-memsipe:0.5.0-alpha1
`PYTHONPATH: :/home/elastest/eds/MemsIPE/../futile/src:/home/elastest/eds/MemsIPE/openmtc/lib:/home/elastest/eds/MemsIPE/../common/*/src:/home/elastest/eds/MemsIPE/../serializers/*/src:/home/elastest/eds/MemsIPE/../openmtc-app/src
PYTHONPATH: :/home/elastest/eds/MemsIPE/openmtc-app/src
DEBUG:__main__:Trying config file location: /home/elastest/eds/MemsIPE/config.json
INFO:__main__:Reading configuration file /home/elastest/eds/MemsIPE/config.json.
INFO:MemsIPE:Registering application as MemsIPE.
DEBUG:OneM2MHTTPClient:Mapping OneM2M request to generic request: OneM2MRequest: originator: CMemsIPE | request_identifier: okMUrJ4ifXdx6ADO | originating_timestamp: None | filter_criteria: None | request_expiration_timestamp: None | result_persistence: None | result_content: None | discovery_result_type: None | group_request_identifier: None | content: AE(path='None', id='None') | to: onem2m | delivery_aggregation: None | role: None | response_type: None | result_expiration_timestamp: None | operation: OneM2MOperation.create | operation_execution_time: None | resource_type: <class 'openmtc_onem2m.model.AE'> | event_category: None
DEBUG:openmtc_onem2m.serializer.util:Encoding result: AE(path='None', id='None') - application/json
DEBUG:OneM2MJsonSerializer:Encoding representation: {'ontologyRef': None, 'App-ID': u'dummy', 'appName': None, 'resourceType': None, 'accessControlPolicyIDs': [], 'labels': [], 'announcedAttribute': None, 'creationTime': None, 'announceTo': None, 'contentSerialization': None, 'pointOfAccess': [u'http://127.0.0.1:5050'], 'requestReachability': False, 'parentID': None, 'lastModifiedTime': None, 'resourceName': u'MemsIPE', 'expirationTime': datetime.datetime(2017, 10, 5, 11, 36, 7, 924940, tzinfo=<openmtc.util.Utc object at 0x7f8270ef52c0>), 'resourceID': None, 'childResource': [], 'AE-ID': None, 'nodeLink': None}
DEBUG:OneM2MHTTPClient:Added request params: {}
ERROR:SimpleFlaskRunner:Error
Traceback (most recent call last):
  File "build/bdist.linux-x86_64/egg/openmtc_app/runner/__init__.py", line 18, in run
    self._run()
  File "build/bdist.linux-x86_64/egg/openmtc_app/flask_runner/__init__.py", line 49, in _run
    self.m2m_app.run(self, self.m2m_ep)
  File "build/bdist.linux-x86_64/egg/openmtc_app/onem2m.py", line 113, in run
    self.register()
  File "build/bdist.linux-x86_64/egg/openmtc_app/onem2m.py", line 191, in register
    registration = self.create_application(app)
  File "build/bdist.linux-x86_64/egg/openmtc_app/onem2m.py", line 286, in create_application
    app = self.mapper.create(path, application)
  File "build/bdist.linux-x86_64/egg/openmtc_onem2m/mapper/__init__.py", line 47, in create
    pc=instance
  File "build/bdist.linux-x86_64/egg/aplus/__init__.py", line 128, in get
    raise self.reason
ConnectionFailed: Error during HTTP request: [Errno -2] Name or service not known. Request was: POST http://frontend:8000/onem2m (0.0327s)
DEBUG:SimpleFlaskRunner:Shutting down
DEBUG:MemsIPE:shutdown handler is finished
DEBUG:MemsIPE:subscriptions deleted
DEBUG:MemsIPE:app deleted
Traceback (most recent call last):
  File "/usr/lib/python2.7/runpy.py", line 174, in _run_module_as_main
    "__main__", fname, loader, pkg_name)
  File "/usr/lib/python2.7/runpy.py", line 72, in _run_code
    exec code in run_globals
  File "/home/elastest/eds/MemsIPE/src/memsipe/__main__.py", line 34, in <module>
    Runner(app, host=host, port=port).run(ep)
  File "build/bdist.linux-x86_64/egg/openmtc_app/runner/__init__.py", line 18, in run

  File "build/bdist.linux-x86_64/egg/openmtc_app/flask_runner/__init__.py", line 49, in _run

  File "build/bdist.linux-x86_64/egg/openmtc_app/onem2m.py", line 113, in run
  File "build/bdist.linux-x86_64/egg/openmtc_app/onem2m.py", line 191, in register
  File "build/bdist.linux-x86_64/egg/openmtc_app/onem2m.py", line 286, in create_application
  File "build/bdist.linux-x86_64/egg/openmtc_onem2m/mapper/__init__.py", line 47, in create

  File "build/bdist.linux-x86_64/egg/aplus/__init__.py", line 128, in get

openmtc.exc.ConnectionFailed: Error during HTTP request: [Errno -2] Name or service not known. Request was: POST http://frontend:8000/onem2m (0.0327s)`

  Could you help us to solve this issue?

Thanks.

**Comment by varungowtham** [comment]
Hi Paco, 

Thank you for pointing out the issue.

Rowshan is now responsible for WP5 from TUB side and therefore, for the EDS related issues. I have pointed her to changes introduced because of the discussion we had in the pull request. She will answer your queries.

Best, 
Varun

**Comment by rowshan** [comment]
Hi,
I am checking it.
Regards,
Rowshan

**Comment by rowshan** [comment]
Hi Francisco,
I have resolved the merge conflict now. I think it should be fine by now.
Regards,
Rowshan

**Comment by varungowtham** [comment]
Link for the discussion thread:
https://github.com/elastest/elastest-device-emulator-service/pull/76

**Comment by rowshan** [comment]
Hi Paco, Can you please close this issue.

**Comment by kickstart530** [comment]
I  am trying to start with docker toolkit and logs on a container indicate the below error. 

Two containers aren't starting up 

1. elastest/etm-proxy:0.9.1

docker logs  
ET_SECURITY ENV VAR: false
Nginx without basic authentication
Generating a 2048 bit RSA private key
.........................................................+++
.......................................+++
writing new private key to '/etc/nginx/ssl/nginx.key'
-----
2018/07/20 01:05:19 [emerg] 12#12: host not found in upstream "edm-elasticsearch" in /etc/nginx/sites-available/nginx-base-location.conf:5
nginx: [emerg] host not found in upstream "edm-elasticsearch" in /etc/nginx/sites-available/nginx-base-location.conf:5
server {
  listen 80 default_server;
  #listen 443 ssl;
  set $security false;

  include sites-available/proxy.conf;
  ssl_certificate /etc/nginx/ssl/nginx.crt;
  ssl_certificate_key /etc/nginx/ssl/nginx.key;

  if ($security = false) {
    set $auth_basic off;
  }
  if ($security = true) {
    set $auth_basic Restricted;
  }
  auth_basic $auth_basic;
  auth_basic_user_file /etc/nginx/conf.d/nginx.htpasswd;

  include sites-available/nginx-base-location.conf;
}

server {
  listen 443 ssl;
  include sites-available/proxy.conf;
  ssl_certificate /etc/nginx/ssl/nginx.crt;
  ssl_certificate_key /etc/nginx/ssl/nginx.key;

  location /logstash/ {
    add_header 'Access-Control-Allow-Origin' '*';
    add_header 'Access-Control-Allow-Methods' 'GET, POST, OPTIONS';
    add_header 'Access-Control-Allow-Headers' 'DNT,User-Agent,X-Requested-With,If-Modified-Since,Cache-Control,Content-Type,Range';
    add_header 'Access-Control-Expose-Headers' 'Content-Length,Content-Range';
    proxy_pass http://etm-logstash:5003/;
  }
}

second container shows the below logs:

$ docker logs 
OpenJDK 64-Bit Server VM warning: If the number of processors is expected to increase from one, then you should configure the number of parallel GC threads appropriately using -XX:ParallelGCThreads=N
OpenJDK 64-Bit Server VM warning: INFO: os::commit_memory(0x0000000705330000, 3133997056, 0) failed; error='Cannot allocate memory' (errno=12)
#
# There is insufficient memory for the Java Runtime Environment to continue.
# Native memory allocation (mmap) failed to map 3133997056 bytes for committing reserved memory.
# An error report file with more information is saved as:
# /usr/share/elasticsearch/hs_err_pid10.log

Clearly something to do with memory allocation, I have memory on my local machine, but looks like It needs some settings on the docker will try to do some research and will let you know if I find something.

**Comment by franciscoRdiaz** [comment]
Hello @kickstart530 

Please, we need more information to help you. Could you answer the following questions?

- Are you using Docker or Docker Toolbox?
- What operating system do you use?
- What are the specifications of your laptop?
- What command are you using to start ElastTest? Please send us the command.
- Could you run the `docker ps -a` command after starting ElasTest and sending us the output?

Finally, please note that the correct place to create these problems is the "elastest/elastestest" repository https://github.com/elastest/elastest

Thank you.

**Comment by kickstart530** [comment]
Hi Francisco,

Sorry i did not notice that tag was device emulator, I reached it via search. Will create it in the elastest in future.
below are the details 
1. I used Docker Toolbox,
2. Windows 10 
3. Laptop has 16GB Ram and 354 GB Free space.
4. I have tried both the below commands.
       $ docker run --rm -v /var/run/docker.sock:/var/run/docker.sock elastest/platform start --server-address="localhost"
and 
$ docker run --rm -v /var/run/docker.sock:/var/run/docker.sock elastest/platform start --server-address=$(docker-machine ip)
5. Below is the command output initially there was  

CONTAINER ID        IMAGE                                COMMAND                  CREATED             STATUS                      PORTS                                                                                                               NAMES
0bca22c64ac5        elastest/eus:0.9.1                   "/bin/sh -c 'exec ja…"   6 hours ago         Up 6 hours                  8040/tcp                                                                                                            07fc4913-03ae-448f-b673-a076aea6e3bf_elastest-eus_1
a562ddb347d3        francescou/docker-compose-ui:1.8.0   "/env/bin/python /ap…"   6 hours ago         Up 6 hours                  0.0.0.0:41515->5000/tcp                                                                                             docker-compose-ui-krpcj2oa16jekrg59s54sdk8tq
8e328049cf1b        elastest/platform                    "python main.py star…"   6 hours ago         Up 6 hours                                                                                                                                      dazzling_chandrasekhar
ea93e84eeaf6        elastest/eus:0.9.1                   "/bin/sh -c 'exec ja…"   12 hours ago        Exited (143) 11 hours ago                                                                                                                       b0b86b56-b895-4b77-8e71-c4ef14737042_elastest-eus_1
e740ffd90a80        francescou/docker-compose-ui:1.8.0   "/env/bin/python /ap…"   12 hours ago        Exited (137) 11 hours ago                                                                                                                       docker-compose-ui-9ht9frpdckh555tsb0nf1sjc9
e8eaa35a4923        elastest/etm-proxy:0.9.1             "/bin/sh -c 'cd /; s…"   12 hours ago        Exited (0) 6 hours ago                                                                                                                          elastest_etm-proxy_1
f1a95a3c82cf        elastest/etm:0.9.1                   "/run.sh"                12 hours ago        Up 6 hours                  8091/tcp                                                                                                            elastest_etm_1
8603eef76860        elastest/etm-filebeat:0.9.1          "/usr/local/bin/dock…"   12 hours ago        Up 6 hours                                                                                                                                      elastest_etm-filebeat_1
729064066a84        elastest/etm-logstash:0.9.1          "/bin/sh -c 'cd /etm…"   12 hours ago        Up 6 hours                  5002-5003/tcp, 0.0.0.0:37500->5000/tcp, 0.0.0.0:37503->5001/tcp, 0.0.0.0:37501->5037/tcp, 0.0.0.0:37502->5044/tcp   elastest_etm-logstash_1
4707a4e556e8        elastest/esm:0.9.1                   "/usr/bin/python3 /a…"   12 hours ago        Up 6 hours                  5000/tcp, 8080/tcp, 37005/tcp                                                                                       elastest_esm_1
251002c63549        elastest/etm-rabbitmq:0.9.1          "docker-entrypoint.s…"   12 hours ago        Up 6 hours                  4369/tcp, 5671-5672/tcp, 15671-15672/tcp, 25672/tcp, 61613/tcp                                                      elastest_etm-rabbitmq_1
36950d8175d2        elastest/edm-mysql:0.9.1             "docker-entrypoint.s…"   12 hours ago        Up 6 hours                  3306/tcp                                                                                                            elastest_edm-mysql_1
3cd00e4c0235        elastest/etm-elasticsearch:0.9.1     "/usr/local/bin/dock…"   12 hours ago        Exited (1) 6 hours ago                                                                                                                          elastest_edm-elasticsearch_1
5e91082a8a85        elastest/eus:0.9.1                   "/bin/sh -c 'exec ja…"   12 hours ago        Exited (143) 11 hours ago                                                                                                                       3b0ce915-17d9-4b82-8727-433584733872_elastest-eus_1
af557747ed68        francescou/docker-compose-ui:1.8.0   "/env/bin/python /ap…"   12 hours ago        Exited (137) 11 hours ago                                                                                                                       docker-compose-ui-27kp7tj42kbkl06sglkhkgievg
b83b48eb74ef        elastest/eus:0.9.1                   "/bin/sh -c 'exec ja…"   12 hours ago        Exited (143) 11 hours ago                                                                                                                       05272567-f659-4694-9d39-534d9c209c59_elastest-eus_1
70b219c715f4        francescou/docker-compose-ui:1.8.0   "/env/bin/python /ap…"   12 hours ago        Exited (137) 11 hours ago                                                                                                                       docker-compose-ui-g4sdtoad760t0nhol2fost6glo
4f7cad508e32        elastest/platform-services:0.9.1     "/bin/sh -c 'exec to…"   12 hours ago        Up 6 hours                                                                                                                                      elastest_platform-services_1
                                                                          NAMES
0bca22c64ac5        elastest/eus:0.9.1                   "/bin/sh -c 'exec ja…"   6 hours ago         Up 6 hours                  8040/tcp                                                                                                            07fc4913-03ae-448f-b673-a076aea6e3bf_elastest-eus_1
a562ddb347d3        francescou/docker-compose-ui:1.8.0   "/env/bin/python /ap…"   6 hours ago         Up 6 hours                  0.0.0.0:41515->5000/tcp                                                                                             docker-compose-ui-krpcj2oa16jekrg59s54sdk8tq
8e328049cf1b        elastest/platform                    "python main.py star…"   6 hours ago         Up 6 hours                                                                                                                                      dazzling_chandrasekhar
ea93e84eeaf6        elastest/eus:0.9.1                   "/bin/sh -c 'exec ja…"   12 hours ago        Exited (143) 11 hours ago                                                                                                                       b0b86b56-b895-4b77-8e71-c4ef14737042_elastest-eus_1
e740ffd90a80        francescou/docker-compose-ui:1.8.0   "/env/bin/python /ap…"   12 hours ago        Exited (137) 11 hours ago                                                                                                                       docker-compose-ui-9ht9frpdckh555tsb0nf1sjc9
e8eaa35a4923        elastest/etm-proxy:0.9.1             "/bin/sh -c 'cd /; s…"   12 hours ago        Exited (0) 6 hours ago                                                                                                                          elastest_etm-proxy_1
f1a95a3c82cf        elastest/etm:0.9.1                   "/run.sh"                12 hours ago        Up 6 hours                  8091/tcp                                                                                                            elastest_etm_1
8603eef76860        elastest/etm-filebeat:0.9.1          "/usr/local/bin/dock…"   12 hours ago        Up 6 hours                                                                                                                                      elastest_etm-filebeat_1
729064066a84        elastest/etm-logstash:0.9.1          "/bin/sh -c 'cd /etm…"   12 hours ago        Up 6 hours                  5002-5003/tcp, 0.0.0.0:37500->5000/tcp, 0.0.0.0:37503->5001/tcp, 0.0.0.0:37501->5037/tcp, 0.0.0.0:37502->5044/tcp   elastest_etm-logstash_1
4707a4e556e8        elastest/esm:0.9.1                   "/usr/bin/python3 /a…"   12 hours ago        Up 6 hours                  5000/tcp, 8080/tcp, 37005/tcp                                                                                       elastest_esm_1
251002c63549        elastest/etm-rabbitmq:0.9.1          "docker-entrypoint.s…"   12 hours ago        Up 6 hours                  4369/tcp, 5671-5672/tcp, 15671-15672/tcp, 25672/tcp, 61613/tcp                                                      elastest_etm-rabbitmq_1
36950d8175d2        elastest/edm-mysql:0.9.1             "docker-entrypoint.s…"   12 hours ago        Up 6 hours                  3306/tcp                                                                                                            elastest_edm-mysql_1
3cd00e4c0235        elastest/etm-elasticsearch:0.9.1     "/usr/local/bin/dock…"   12 hours ago        Exited (1) 6 hours ago                                                                                                                          elastest_edm-elasticsearch_1
5e91082a8a85        elastest/eus:0.9.1                   "/bin/sh -c 'exec ja…"   12 hours ago        Exited (143) 11 hours ago                                                                                                                       3b0ce915-17d9-4b82-8727-433584733872_elastest-eus_1
af557747ed68        francescou/docker-compose-ui:1.8.0   "/env/bin/python /ap…"   12 hours ago        Exited (137) 11 hours ago                                                                                                                       docker-compose-ui-27kp7tj42kbkl06sglkhkgievg
b83b48eb74ef        elastest/eus:0.9.1                   "/bin/sh -c 'exec ja…"   12 hours ago        Exited (143) 11 hours ago                                                                                                                       05272567-f659-4694-9d39-534d9c209c59_elastest-eus_1
70b219c715f4        francescou/docker-compose-ui:1.8.0   "/env/bin/python /ap…"   12 hours ago        Exited (137) 11 hours ago                                                                                                                       docker-compose-ui-g4sdtoad760t0nhol2fost6glo
4f7cad508e32        elastest/platform-services:0.9.1     "/bin/sh -c 'exec to…"   12 hours ago        Up 6 hours                                                                                                                                      elastest_platform-services_1

Note: I am able to run it on a mac after a few tries.
